### PR TITLE
Create molecular-plant-pathology.csl

### PR DESCRIPTION
**Pull request by libora**
Zotero forum discussion:
https://forums.zotero.org/discussion/51058/style-request-molecular-plant-pathology/

### CODE REVIEW

File: molecular-plant-pathology.csl [hunk 1]
```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US">
+  <info>
+    <title>Molecular plant pathology</title>
```
Comment (adam3smith) [inline molecular-plant-pathology.csl]: we title case journal names: Molecular Plant Pathology

File: molecular-plant-pathology.csl [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter=", " delimiter-precedes-last="never" initialize-with="., "/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
```
Comment (adam3smith) [inline molecular-plant-pathology.csl]: I'd remove the translator here (or add it to the author macro above)--either way, the substitute chain for author and author-short should almost always be identical.

File: molecular-plant-pathology.csl [hunk 3]
```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US">
+  <info>
+    <title>Molecular plant pathology</title>
+    <id>http://www.zotero.org/styles/molecular-plant-pathology</id>
+    <link href="http://www.zotero.org/styles/molecular-plant-pathology" rel="self"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/%28ISSN%291364-3703/homepage/ForAuthors.html" rel="documentation"/>
```
Comment (adam3smith) [inline molecular-plant-pathology.csl]: which style did you base this on (if any)? We'd normally want a template link.

File: molecular-plant-pathology.csl [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+          </group>
+        </else-if>
+        <else>
+          <group suffix=".">
+            <text macro="title" prefix=" "/>
+            <text macro="editor" prefix=" "/>
+          </group>
+          <group prefix=" " suffix=".">
+            <text variable="container-title" form="short" font-style="italic"/>
+            <group prefix=" ">
```
Comment (adam3smith) [inline molecular-plant-pathology.csl]: remove the group here and in line 174 and move the prefix to the single variable they contain -- there's no need for groups with only a single element.

**Comment by adam3smith** [comment]
OK, quick fixes, should only take you a couple of minutes. Thanks!

File: molecular-plant-pathology.csl (hunk 3 above)
Comment (libora) [inline molecular-plant-pathology.csl]: I do not remember which style I use as base for this template, sorry.

**Comment by libora** [comment]
Its done.

**Comment by adam3smith** [comment]
OK, thanks!

**Pull request merged**